docs/whats-new.md

```diff
@@ -1,335 +1,237 @@
-| --------- | ----------------- | ------------------ | --------------------------------------------------------------------------------------- |
-| Default   | `ribs`            | `pyribs`           | Basic pyribs package.                                                                   |
-| Visualize | `ribs[visualize]` | `pyribs-visualize` | Adds visualization dependencies, currently just Matplotlib.                             |
-| All       | `ribs[all]`       | `pyribs-all`       | Installs dependencies for all extra distributions (currently just the Visualize extra). |
-
-## Goal 1: Implement New Algorithms
-
-Our first major goal in pyribs v0.5.0 has been to implement an array of new
-algorithms via new emitters and schedulers.
-
-### Flexible CMA-ME and CMA-MAE with EvolutionStrategyEmitter
-
-We introduce the {class}`~ribs.emitters.EvolutionStrategyEmitter` which replaces
-the earlier `ImprovementEmitter`, `RandomDirectionEmitter`, and
-`OptimizingEmitter`. This emitter may be used in both
-[CMA-ME](https://arxiv.org/abs/1912.02400) and
-[CMA-MAE](https://arxiv.org/abs/2205.10752). The behaviors of these earlier
-emitters may be replicated by selecting an appropriate _ranker_ from the
-{mod}`ribs.emitters.rankers` module. For example:
-
-```python
-EvolutionStrategyEmitter(archive, ..., ranker="2imp")  # Equivalent to ImprovementEmitter.
-EvolutionStrategyEmitter(archive, ..., ranker="imp")  # Single-stage improvement ranking as is used in CMA-MAE.
-EvolutionStrategyEmitter(archive, ..., ranker="2rd")  # Two-stage random direction ranking.
-EvolutionStrategyEmitter(archive, ..., ranker="obj")  # Objective ranking as was done in OptimizingEmitter.
-```
-
-{class}`~ribs.emitters.EvolutionStrategyEmitter` also supports evolution
-strategies other than CMA-ES, thus enabling it to implement scalable variants of
-CMA-ME and CMA-MAE described in
-[Tjanaka 2022](https://arxiv.org/abs/2210.02622). These evolution strategies are
-available in the {mod}`ribs.emitters.opt` module. For example:
+# What's New in v0.7.0
 
-```python
-EvolutionStrategyEmitter(archive, ..., es="sep_cma_es")  # sep-CMA-ES instead of CMA-ES.
-```
-
-### DQD Algorithms with GradientArborescenceEmitter
+The updates in v0.7.0 centered around making the archives more flexible and
+adding new algorithmic features. Below we describe some of the key changes. For
+the full list of changes, please refer to our [History page](./history).
 
-We have added a {class}`~ribs.emitters.GradientArborescenceEmitter` which
-supports [DQD algorithms](https://arxiv.org/abs/2106.03894). For usage examples,
-see the tutorial {doc}`tutorials/tom_cruise_dqd`.
+## More Flexible Archives
 
-### Custom Initial Solutions in Emitters
+We refactored our archives to build on a data structure we call an
+{class}`~ribs.archives.ArrayStore`, which stores archive data in several
```

Comment:
It might be good to say what the fields in ArrayStore would be since it is not immediately clear

Reply:
Added description of fields. Thanks!